### PR TITLE
feat(payments): PAYPAL-823 Add new banner on the cart page

### PR DESCRIPTION
**Pull request by helga1507**
#### What?
New PayPal credit banner on the cart page above the checkout button

#### Tickets / Documentation

Add links to any relevant tickets and documentation.
 [Ticket](https://jira.bigcommerce.com/browse/PAYPAL-823)

#### Screenshots (if appropriate)

Screenshot 2020-11-11 at 18 46 43

### CODE REVIEW

**Review by bc-alexsaiannyi — APPROVED** [review]

**Comment by BC-tymurbiedukhin** [comment]
@helga1507 Could you please add a changelog item?

**Comment by helga1507** [comment]
@BC-tymurbiedukhin Sure

File: CHANGELOG.md [hunk 1]
```diff
@@ -643,3 +644,6 @@ No actual PR updates. Fixes config values.
 - Hide Gift Certificates when the setting is disabled in the control panel [#914](https://github.com/bigcommerce/cornerstone/pull/914) & [#916](https://github.com/bigcommerce/cornerstone/pull/916)
 - Fix an issue with a close button on the quick search modal on mobile [#918](https://github.com/bigcommerce/cornerstone/pull/918)
 - Adding CHANGELOG.md [#919](https://github.com/bigcommerce/cornerstone/pull/919)
+
+
+[]: https://github.com/bigcommerce/cornerstone/pull/1901
```
Comment (BC-tymurbiedukhin) [inline CHANGELOG.md:649]: Could we remove these lines?

Comment (helga1507) [reply CHANGELOG.md:649]: Oh, sure.
Thank you!

**Comment by BC-tymurbiedukhin** [comment]
LGTM

**Review by BC-tymurbiedukhin — APPROVED** [review]

File: templates/pages/cart.html [hunk 2]
```diff
@@ -24,6 +24,8 @@
                 {{> components/cart/totals}}
             </div>
 
+            {{{region name="banner_below_price"}}}
```
Comment (BC-ACherednichenko) [inline templates/pages/cart.html:27]: I would recommend being more specific with the name on a cart page (as it is not a generic region like in the header), for instance, `cartpage_banner_below_totals`

Comment (helga1507) [reply templates/pages/cart.html:27]: Okay. Thank you!

**Pull request merged**